locales/en/app.ftl

```diff
@@ -459,3 +459,305 @@ get-your-report-and-sign-up= Get your report and sign up for alerts.
+  is to use unique passwords for each one.
+
+
+ba-next-step-3 = Get help creating better passwords and keeping them safe.
+ba-next-step-blurb-3 =
+  Use a password manager to create strong, unique passwords. Password managers securely store all your 
+  logins so you can access them across all your devices.
+
+faq1 = I don’t recognize this company or website. Why am I in this breach?
+faq2 = Why did it take so long to notify me of this breach?
+faq3 = How do I know this is a legitimate email from {-product-name}?
+
+
+new-breaches-found =
+  { $breachCount ->
+     *[one] { $breachCount } NEW BREACH FOUND
+      [other] { $breachCount } NEW BREACHES FOUND
+  }
+
+sign-up-headline-1 = Get ongoing alerts with a {-brand-fxa}.
+sign-up-for-alerts = Sign up for Alerts
+account-not-required = {-brand-name} browser not required for a {-brand-fxa}. You may receive info about {-brand-Mozilla} services.
+
+
+get-alerted = Get alerted about new breaches.
+
+was-your-info-exposed = Was your info exposed in the { $breachName } data breach?
+find-out-if = Find out if your data was exposed in this breach.
+
+fb-not-comp = This email did not appear in the { $breachName } breach.
+fb-comp = This email appeared in { $breachCount } known data breaches, including { $breachName }. 
+fb-comp-only = This email appeared in the { $breachCount } breach.
+
+other-breaches-found = However, it did appear in 
+  { $breachCount ->
+    [one] { $breachCount } other breach.
+   *[two] { $breachCount } other breaches. 
+  }
+
+fb-comp-and-others = This email appeared in { $breachCount } known data breaches, including { $breachName }.
+no-other-breaches-found = No other breaches found from a basic search.
+
+no-results-blurb = Sorry, that breach is not in our database.
+all-breaches-headline = All breaches in {-product-name}
+search-breaches = Search Breaches
+
+# Tells the user which breach types are currently being displayed. In page, it would look like "Showing: All Breaches"
+currently-showing = Showing:
+all-breaches= All Breaches
+
+
+## Updated error messages
+
+error-bot-headline = Searches temporarily suspended
+error-bot-blurb = We’re worried you might be a bot because you searched 
+  several email addresses in a short time period. For now, you’re blocked 
+  from new searches. You can try again later.
+
+error-csrf-headline = Session timed out
+error-csrf-blurb = Select your browser’s back button, reload the page, and try again.
+
+error-invalid-unsub = How to unsubscribe from {-product-name} alerts
+error-invalid-unsub-blurb = You’ll need to unsubscribe from one of the 
+  emails {-product-name} sent you. Check your inbox for messages from 
+  {-brand-team-email}. Select the unsubscribe link at the bottom of the email.
+
+
+login-link-pre = Have an account?
+login-link = Log in
+
+
+# This string follows a large numeral that indicates the total number of email address a user has signed up for monitoring.
+# For instance, "7 Email addresses being monitored"
```

Comment:
```
# This string is displayed under a large numeral that indicates the total number
# of email address a user has signed up for monitoring. Don't add $emails to
# your localization, because it would result in the number showing twice.
```